app/views/course_request/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<!-- isn't used rn -->
+<%= link_to "Tutor Matching", course_request_index_path %>
+<h1> Course Request </h1>
+<b1><%= @crequest.course_name_full%> </b1> <br>
+#<%= CourseRequest.all.inspect %>
+
+
```

Comment:
Should this be here? It doesn't look bootstrapped or finished, not to mention the comment at the top

Reply:
the show file needs to be present or else the controller acts up, but it isn't actually supposed to be accessed